zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/initialize/PlatformDefaultEntities.java

```diff
@@ -54,15 +54,16 @@ private static void setupReadOnlyAdminRole(final IdentitySetupRecord setupRecord
           resourceType.getSupportedPermissionTypes().stream()
               .filter(PermissionType::isReadPermission)
               .collect(Collectors.toSet());
-
-      setupRecord.addAuthorization(
-          new AuthorizationRecord()
-              .setOwnerType(AuthorizationOwnerType.ROLE)
-              .setOwnerId(readOnlyAdminRoleId)
-              .setResourceType(resourceType)
-              .setResourceMatcher(WILDCARD.getMatcher())
-              .setResourceId(WILDCARD.getResourceId())
-              .setPermissionTypes(readBasedPermissions));
+      if (!readBasedPermissions.isEmpty()) {
```

Comment:
❓ Why this check ?

Reply:
I tend to agree with @nicpuppa (but don't have a strong argument therefore) 😄